packages/nimble-components/src/wafer-map/tests/wafer-map.stories.ts

```diff
@@ -119,12 +119,16 @@ const metadata: Meta<WaferMapArgs> = {
                 color: red;
                 font: var(${bodyFont.cssCustomProperty});
             }
+            #wafer-map {
+                width: 100%;
+                height: 500px;
```

Comment:
Is there a way to have the component host html page inside the storybook cover all of the height and not just al of the width? when setting the height to 100% the component shrinks and is barely visible

Reply:
We recently started using [css aspect ratio](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio) to force certain components to be square if the user only provides a width or a height. I'm not sure if it would help in this situation, just an idea to consider. You can search the repo for `aspect-ratio` to see an example for the spinner component.

Reply:
I put an 16 / 9 aspect, do you think it's better this way?

Reply:
Hmm, I was thinking it would be 1 / 1 to force it to be square and fill the width given to it. I don't see a reason to force extra whitespace on each side (though it does look like it's being displayed on a movie theater screen, which is kinda cool 😆).

Reply:
With this change we wanted to make sure that the client can use all the space he has, that is why we made the canvas as big as it can be so when zooming it will not be confined to the initial square, so, we want to support any aspect ratio they want to have and here it was just a demonstration of that ability